study-builder/config/brugada-config.json.ctmpl

```diff
@@ -0,0 +1,36 @@
+{{$environment := env "ENVIRONMENT"}}
+{{$version := env "VERSION"}}
+{{with $response := vault (printf "secret/pepper/%s/%s/brugada/conf" $environment $version)}}
+{{with $conf := $response.Data}}
+
+{
```

Comment:
we need a entry for analyticsToken (google analytics token)

Reply:
analyticsToken is being added
<img width="807" alt="image" src="https://user-images.githubusercontent.com/71830706/135334987-4134170e-8c64-4f16-8701-85525a99cd75.png">